app/scrapers/football_scraper.py

```diff
@@ -0,0 +1,315 @@
+import json
+import re
+from datetime import datetime, timedelta
+
+import bs4
+import httpx
+from bs4 import BeautifulSoup
+from config.logging import setup_logging
+from models.schemas import CalendarEvent, EventDate
+
+logger = setup_logging(__name__)
+
+
+class FootballScraper:
+    """
+    Scraper for Maccabi Tel Aviv football matches
+    """
+
+    def __init__(self):
+        self.season_link_unformatted: str = "https://www.maccabi-tlv.co.il/%d7%9e%d7%a9%d7%97%d7%a7%d7%99%d7%9d-%d7%95%d7%aa%d7%95%d7%a6%d7%90%d7%95%d7%aa/%d7%94%d7%a7%d7%91%d7%95%d7%a6%d7%94-%d7%94%d7%91%d7%95%d7%92%d7%a8%d7%aa/%d7%aa%d7%95%d7%a6%d7%90%d7%95%d7%aa/?season={season_number}#content"
```

Comment:
Move to consts

Reply:
למה? זה אומר לתחזק const לכל סוג של scarper.
 לא יותר נוח שכמה שיותר מידע נמצא בקובץ אחד?

Reply:
אפשר לשים את כל הconsts בקובץ אחד, ואם זה עדיין בעייתי אפשר פה בקובץ למעלה.

זו פשוט קונבנציה שאם יש דברים שהם כמו מספרי קסם, לינקים או כל דבר אחד שהוא קבוע אז ישר אפשר לחפש אתו במקום הקבוע, מעבר לזה תסתכל על זה למשל, זה נראה לי קריא הרבה יותר:

self._upcoming_matches_url = MACCABI_TLV_SITE_SEASONS_UNFORMATTED_URL

בתכלס לא אפכפת לי מה הלינק פה